webui/react/src/pages/ProjectDetails.tsx

```diff
@@ -180,7 +170,10 @@ const ProjectDetails: React.FC = () => {
                 <ProjectActionDropdown
                   project={project}
                   showChildrenIfEmpty={false}
-                  workspaceArchived={workspace?.archived}
+                  workspaceArchived={Loadable.match(workspace, {
+                    Loaded: (w: Workspace) => w?.archived,
+                    NotLoaded: () => false,
+                  })}
```

Comment:
with the suggestion above, this line could remain unchanged.

Reply:
✅